plugins/woocommerce-admin/client/products/product-form-actions.tsx

```diff
@@ -42,12 +42,23 @@ export const ProductFormActions: React.FC = () => {
 		};
 	};
 
+	const maybeSetRegularPrice = ( product: Product ) => {
+		if (
+			product.regular_price === undefined &&
```

Comment:
Should we also check if `regular_price` is an empty string?
As this will break as soon as the user edits the list price, but clears it.

Reply:
Nice catch! This would be redundant considering Maikel's [PR here](https://github.com/woocommerce/woocommerce/pull/34658). I removed the method `maybeSetRegularPrice` in commit [0e65863](https://github.com/woocommerce/woocommerce/pull/34558/commits/0e6586341082a7e33a22a24602fa9b4e89fe39d9)